README.md

```diff
@@ -42,13 +42,15 @@ export class AppModule {}
 
 ```ts
 import { DatepickerOptions } from 'ng2-datepicker';
+import * as frLocale from 'date-fns/locale/fr';
```

Comment:
is there a reason why we should have `fr` hardcoded?

Reply:
@jkuri 
No, it's just an example. It could be any other language code.

Reply:
@jkuri 
Could you please accept my PR ?